apps/src/lib/kits/maker/toolkit.js

```diff
@@ -164,8 +164,9 @@ function confirmSupportedBrowser() {
  * @returns {Promise.<MakerBoard>}
  */
 function getBoard() {
+  const makerBoardAPI = getAppOptions().level.makerlabEnabled;
```

Comment:
Just for my understanding: this reads a bit weird to me - `makerlabEnabled` sounds like it should return a boolean value, but looks like it's instead a constant that we compare to value of `MB_API` to determine if we are using MicroBit or CP? What value is returned by `makerlabEnabled`?

Reply:
Yes - I agree it sounds like it should return a boolean. I believe it is because we only used to support the CircuitPlayground and now we support 2 different boards. `makerlabEnabled` returns `microbit` if a micro:bit is assigned or `true` if a Circuit Playground is connected. I'll make a note that we should probably refactor the name and the return for a CP.